src/app/clusters/closure-dimension-server/closure-dimension-cluster-logic.cpp

```diff
@@ -0,0 +1,469 @@
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetUnitRange(DataModel::Nullable<Structs::UnitRangeStruct::Type> & unitRange)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mConformance.HasFeature(Feature::kUnit), CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    unitRange = mState.unitRange;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetLimitRange(Structs::RangePercent100thsStruct::Type & limitRange)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mConformance.HasFeature(Feature::kLimitation), CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    limitRange = mState.limitRange;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetTranslationDirection(TranslationDirectionEnum & translationDirection)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mConformance.HasFeature(Feature::kTranslation), CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    translationDirection = mState.translationDirection;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetRotationAxis(RotationAxisEnum & rotationAxis)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mConformance.HasFeature(Feature::kRotation), CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    rotationAxis = mState.rotationAxis;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetOverflow(OverflowEnum & overflow)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError((mConformance.HasFeature(Feature::kRotation) || mConformance.HasFeature(Feature::kMotionLatching)),
+                        CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    VerifyOrReturnError(mConformance.supportsOverflow, CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    overflow = mState.overflow;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetModulationType(ModulationTypeEnum & modulationType)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    VerifyOrReturnError(mConformance.HasFeature(Feature::kModulation), CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    modulationType = mState.modulationType;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetFeatureMap(Attributes::FeatureMap::TypeInfo::Type & featureMap)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    featureMap = mConformance.featureMap;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ClusterLogic::GetClusterRevision(Attributes::ClusterRevision::TypeInfo::Type & clusterRevision)
+{
+    VerifyOrReturnError(mInitialized, CHIP_ERROR_INCORRECT_STATE);
+    clusterRevision = kClusterRevision;
+    return CHIP_NO_ERROR;
+}
+
+Status ClusterLogic::HandleSetTargetCommand(Optional<Percent100ths> position, Optional<TargetLatchEnum> latch,
+                                            Optional<Globals::ThreeLevelAutoEnum> speed)
+{
+    VerifyOrReturnError(mInitialized, Status::Failure);
```

Comment:
I'm not sure if this should return an error or die since the application isn't in a valide state.
I think it should be a VerifyOrDie in this case.